src/plugins/telemetryTable/TelemetryTable.js

```diff
@@ -442,11 +444,18 @@ export default class TelemetryTable extends EventEmitter {
   }
 
   sortBy(sortOptions) {
-    this.tableRows.sortBy(sortOptions);
+    this.sortOptions = sortOptions;
```

Comment:
The TelemetryTableConfiguration class exists to try and abstract away some of the boiletplate involved in reconciling in-memory and persisted configuration. Maybe we could move some of this boilerplate there?

eg.
```
getSortOptions() {
    return this.#sortOptions || 
        this.getConfiguration().sortOptions ||
        {
              key: this.openmct.time.getTimeSystem().key,
              direction: ORDER.DESCENDING
        }
}
setSortOptions(sortOptions) {
   this.sortOptions = sortOptions;

   if (this.openmct.editor.isEditing()) {
      let configuration = this.configuration.getConfiguration();
      configuration.sortOptions = sortOptions;
      this.configuration.updateConfiguration(configuration);
    }
}
```